infra/terraform/environments/alpha/main.tf

```diff
@@ -86,6 +86,8 @@ module "container_registry" {
 
 module "data_access" {
```

Comment:
Thanks for your review @kerin! I see what you mean. Not sure although.

It's still not 100% clear how I'll structure the code (and I'll probably move few things down the line as I go) but I was planning to have all the resources related to the data access in this module, e.g. also the lambda functions to perform atomic operations like, `create_iam_role`, `delete_iam_role`, etc...which may make sense also outside a `webhook_api` module and which we may manually trigger and/or from an UI later at some point.